js/services/vCardProperties.js

```diff
@@ -124,14 +127,17 @@ angular.module('contactsApp')
 		'X-SOCIALPROFILE': {
 			multiple: true,
 			readableName: t('contacts', 'Social network'),
-			template: 'text',
+			template: 'username',
 			defaultValue: {
 				value:[''],
 				meta:{type:['facebook']}
 			},
 			options: [
 				{id: 'FACEBOOK', name: 'Facebook'},
+				{id: 'INSTAGRAM', name: 'Instagram'},
+				{id: 'LINKEDIN', name: 'LinkedIn'},
 				{id: 'TWITTER', name: 'Twitter'}
+
```

Comment:
Should be the same options of socialsharing-app (https://github.com/nextcloud/socialsharing) ... So Google+ is a good idea 👍